trp_use_case/security/ir_rule.xml

```diff
@@ -7,7 +7,8 @@
             <field name="model_id" ref="trp_use_case.model_use_case_collection"/>
             <field name="groups" eval="[(4, ref('trp_use_case.group_external_use_case'))]"/>
             <field name="domain_force">[
-                ('partner_id', 'in', [x.id for x in user.external_user_partner_ids])]</field>
+                '|', ('partner_id', 'in', [x.id for x in user.external_user_partner_ids]),
+                ('message_follower_ids', 'in', [x.id for x in user.external_user_partner_ids])]</field>
```

Comment:
that's a very good idea! This code is inherited from 6.1, by now you can write the expressions simpler as for example `('partner_id', 'in', user.external_user_partner_ids.ids)`